exchanges/kraken/kraken_websocket.go

```diff
@@ -599,19 +599,23 @@ func (k *Kraken) wsProcessOpenOrders(ownOrders interface{}) error {
 // allowing correlation between subscriptions and returned data
```

Comment:
I was adding them very early on before running into the issues that we will need a connections manager or something. And then it got sidetracked so I just left it in there since it'll still be useful for the next person implementing it.